adov_flutter/lib/screens/add_talk/add_talk.dart

```diff
@@ -0,0 +1,294 @@
+import 'package:adov_flutter/models/talk.dart';
+import 'package:adov_flutter/widgets/date_time_picker.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:toast/toast.dart';
+
+TextEditingController titleController = TextEditingController();
+TextEditingController roomController = TextEditingController();
+TextEditingController descriptionController = TextEditingController();
+DateTime selectedDate = DateTime(2020);
+DateTime selectedTime = DateTime(2020);
+
+class AddTalkFormState extends State<AddTalk> {
+  // Create a global key that uniquely identifies the Form widget
+  // and allows validation of the form.
+
+  final _formKey = GlobalKey<FormState>();
+  final _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  Widget build(BuildContext context) {
+    // Build a Form widget using the _formKey created above.
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
+        title: Text('Add a Talk'),
+      ),
+      body: SingleChildScrollView(
+        scrollDirection: Axis.vertical,
+        child: GestureDetector(
+          onTap: () {
+            FocusScopeNode currentFocus = FocusScope.of(context);
+            if (!currentFocus.hasPrimaryFocus) {
+              currentFocus.unfocus();
+            }
+          },
+          child: Form(
+            key: _formKey,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
```

Comment:
Isto é para dizer que é uma lista de widgets o.O

Reply:
sim mas não é necessário, no Dart 2 não é pelo menos, só está aí pq StackOverFlow is life